src/autopas/selectors/tuningStrategy/GaussianModel/GaussianProcess.h

```diff
@@ -81,6 +81,14 @@ class GaussianProcess {
    */
   [[nodiscard]] size_t numEvidence() const { return _inputs.size(); }
 
+  /**
+   * Get all currently stored evidence
+   * @return pair of inputs and outputs
+   */
+  [[nodiscard]] const std::pair<std::vector<Vector>, Eigen::VectorXd> getEvidence() const {
```

Comment:
```suggestion
  [[nodiscard]] std::pair<std::vector<Vector>, Eigen::VectorXd> getEvidence() const {
```
This `const` does not do anything.